Add Sam's Blog

EStyli opened the pull request:
Edited blog to ~912 words, originally 1352 words

EStyli committed:
Add Sam's Blog

AnilMistry reviewed (changes requested):
Good work, just a few things to update then happy to approve.

EStyli committed:
Sam Blog Edits

added quoation marks where needed, removed spaces for links to work, shotened title, added Sam's image and cropped main image and made other minor edits.

EStyli reviewed (commented):
Review Sam Blog Edits
added quoation marks where needed, removed spaces for links to work, shotened title, added Sam's image and cropped main image and made other minor edits.

dangerdika reviewed (approved):
Loads well, cant find any typos, all problems @AnilMistry commented on have been resolved. Thank you @EStyli, merging this now!
